modules/postgresql/src/main/java/org/testcontainers/containers/PostgreSQLContainer.java

```diff
@@ -73,4 +80,23 @@ public SELF withPassword(final String password) {
         this.password = password;
         return self();
     }
+
+    @Override
+    protected void waitUntilContainerStarted() {
+        String regEx = ".*database system is ready to accept connections.*\\s";
+        logger().info("Waiting for database to log '{}'", regEx);
+
+        WaitingConsumer waitingConsumer = new WaitingConsumer();
+        this.followOutput(waitingConsumer);
+
+
+        Predicate<OutputFrame> waitPredicate = outputFrame ->
+                outputFrame.getUtf8String().matches(regEx);
```

Comment:
Yeah, I'd prefer having getUtf8String return an empty string too.  Otherwise I think we end up with null checks in all kinds of places, like https://github.com/testcontainers/testcontainers-java/blob/master/docs/usage/options.md#waiting-for-container-output-to-contain-expected-content.